tests/terraform/create_image.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create HDD image that boots automatically and add a second
+#          interface config file.
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use warnings;
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+
+    # Don't wait for user to press enter, boot the system automatically
+    assert_script_run('sed -i s/GRUB_TIMEOUT=-1/GRUB_TIMEOUT=0/ /etc/default/grub');
+    assert_script_run('grub2-mkconfig -o /boot/grub2/grub.cfg');
+
+    # Allow having a second NIC configurable
+    assert_script_run('cp /etc/sysconfig/network/ifcfg-eth0 /etc/sysconfig/network/ifcfg-eth1');
```

Comment:
no point to do it here , you have no idea what name NICs will have ( eth0 vs ens0 )

Reply:
It's normally eth0 eth0 in this kind of SLE images, but I will test it also with opensuse to doublecheck

Reply:
You should delete `/etc/udev/rules.d/70-persistent-net.rules` to have properly configured `eth0` during first boot.

Reply:
as far as I know it is not image related but qemu setup , and I can tell you that in o3 environment it is ens0 , this is why iface() function was introduced in wicked tests :)

Reply:
Just tested it, it's ens3 and ens4 for Tumbleweed and eth0/eth1 for SLE/Leap15